Jenkinsfile

```diff
@@ -141,8 +146,8 @@ pipeline {
             steps {
                 onStage()
                 colourText('info', "Building Python package.")
-                unstash name: 'Checkout'
-                unstash name: 'venv'
+                // unstash name: 'Checkout'
+                // unstash name: 'venv'
```

Comment:
Craig's notes said to remove the unstash in the "Post" section. I can't find that, but think it may be this one. So I've just commented out.